lib/tools/system-calls.js

```diff
@@ -406,11 +410,11 @@ systemCallMethods.adbExec = async function adbExec (cmd, opts = {}) {
       const args = [...this.executable.defaultArgs, ...cmd];
       log.debug(`Running '${this.executable.path} ` +
         (args.find((arg) => /\s+/.test(arg)) ? util.quote(args) : args.join(' ')) + `'`);
-      let {stdout} = await exec(this.executable.path, args, opts);
+      let {stdout, stderr, code} = await exec(this.executable.path, args, opts);
       // sometimes ADB prints out weird stdout warnings that we don't want
       // to include in any of the response data, so let's strip it out
       stdout = stdout.replace(LINKER_WARNING_REGEXP, '').trim();
-      return stdout;
+      return opts.outputFormat === 'full' ? {stdout, stderr, code} : stdout;
```

Comment:
can we have constants instead of magic strings?
also `code` is always going to be equal zero there (otherwise an exception is thrown)

Reply:
Separated `full` and `stdout` strings into consts within the function's scope. I saw both within-function and global consts in system-calls.js; let me know if you'd prefer these to be global.